plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -578,6 +579,7 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
             cleanExtraOutputFiles(session, writtenFiles.build());
         }
 
+        appendFiles.set(PRESTO_QUERY_ID_NAME, session.getQueryId());
```

Comment:
Since it's a new feature, we should use `trino_query_id`.

Reply:
cc @electrum

Reply:
Yeah that seems like a good idea. I will add a constant in for that and if we add the constant in another PR, I'll use the updated constant. 👍